listing.amp.html

```diff
@@ -7,79 +7,220 @@
     <title>AMP Lovethesign</title>
 
     <!-- need to point to original resource -->
-    <link rel="canonical" href="#">
+    <link rel="canonical" href="listing.amp.html">
     <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
 
     <script async custom-element="amp-list" src="https://cdn.ampproject.org/v0/amp-list-0.1.js"></script>
     <script async custom-template="amp-mustache" src="https://cdn.ampproject.org/v0/amp-mustache-0.2.js"></script>
```

Comment:
Please remove amp-mustache dependencies.

Reply:
There is an high chance that mustace will be implemented soon, I commented that line.